tests/arch/xilinx/tribuf.ys

```diff
@@ -8,5 +8,7 @@ equiv_opt -assert -map +/xilinx/cells_sim.v -map +/simcells.v synth_xilinx # equ
 design -load postopt # load the post-opt design (otherwise equiv_opt loads the pre-opt design)
 cd tristate # Constrain all select calls below inside the top module
 # TODO :: Tristate logic not yet supported; see https://github.com/YosysHQ/yosys/issues/1225
```

Comment:
This is obviously untrue now :p

Reply:
addressed in 436fea9e6990c66369d7c30b571920ae115efb44